src/snapred/backend/dao/StateId.py

```diff
@@ -1,4 +1,9 @@
+import hashlib
+import json
 from dataclasses import dataclass
+from typing import ClassVar
+
+from pydantic import Field
```

Comment:
It doesn't look like these new imports are being used

Reply:
Removed.